["```py\n203.0.113.12\n```", "```py\n**$ ip addr**\n**1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN**\n **link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00**\n **inet 127.0.0.1/8 scope host lo**\n **valid_lft forever preferred_lft forever**\n**2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP qlen 1000**\n **link/ether b8:27:eb:5d:7f:ae brd ff:ff:ff:ff:ff:ff**\n **inet 192.168.0.4/24 brd 192.168.0.255 scope global eth0**\n **valid_lft forever preferred_lft forever**\n\n```", "```py\n192.168.0.186/24\n```", "```py\n255.255.255.0\n```", "```py\n255.0.0.0       = 11111111 00000000 00000000 00000000 = /8\n255.192.0.0     = 11111111 11000000 00000000 00000000 = /10\n255.255.255.0   = 11111111 11111111 11111111 00000000 = /24\n255.255.255.240 = 11111111 11111111 11111111 11110000 = /28\n```", "```py\n**$ nslookup python.org**\n**Server:         192.168.0.4**\n**Address:        192.168.0.4#53**\n\n**Non-authoritative answer:**\n**Name:   python.org**\n**Address: 104.130.43.121**\n\n```", "```py\n**>>> import smtplib**\n**>>> smtplib.SMTP('127.0.0.1', port=66000)**\n\n```", "```py\nTraceback (most recent call last):\n  File \"<stdin>\", line 1, in <module>\n  File \"**/usr/lib/python3.4/smtplib.py**\", line 242, in __init__\n    (code, msg) = self.connect(host, port)\n  File \"**/usr/lib/python3.4/smtplib.py**\", line 321, in connect\n    self.sock = self._get_socket(host, port, self.timeout)\n  File \"**/usr/lib/python3.4/smtplib.py**\", line 292, in _get_socket\n    self.source_address)\n  File \"**/usr/lib/python3.4/socket.py**\", line 509, in create_connection\n    raise err\n  File \"**/usr/lib/python3.4/socket.py**\", line 500, in create_connection\n    sock.connect(sa)\nConnectionRefusedError: [Errno 111] Connection refused\n```", "```py\nimport sys, urllib.request\n\ntry:\n    rfc_number = int(sys.argv[1])\nexcept (IndexError, ValueError):\n    print('Must supply an RFC number as first argument')\n    sys.exit(2)\n\ntemplate = 'http://www.ietf.org/rfc/rfc{}.txt'\nurl = template.format(rfc_number)\nrfc_raw = urllib.request.urlopen(url).read()\nrfc = rfc_raw.decode()\nprint(rfc)\n```", "```py\n**$ python RFC_downloader.py 2324 | less**\n\n```", "```py\nimport sys, socket\n\ntry:\n    rfc_number = int(sys.argv[1])\nexcept (IndexError, ValueError):\n    print('Must supply an RFC number as first argument')\n    sys.exit(2)\n\nhost = 'www.ietf.org'\nport = 80\nsock = socket.create_connection((host, port))\n\nreq = (\n    'GET /rfc/rfc{rfcnum}.txt HTTP/1.1\\r\\n'\n    'Host: {host}:{port}\\r\\n'\n    'User-Agent: Python {version}\\r\\n'\n    'Connection: close\\r\\n'\n    '\\r\\n'\n)\nreq = req.format(\n    rfcnum=rfc_number,\n    host=host,\n    port=port,\n    version=sys.version_info[0]\n)\nsock.sendall(req.encode('ascii'))\nrfc_raw = bytearray()\nwhile True:\n    buf = sock.recv(4096)\n    if not len(buf):\n        break\n    rfc_raw += buf\nrfc = rfc_raw.decode('utf-8')\nprint(rfc)\n```", "```py\nHTTP/1.1 200 OK\nDate: Thu, 07 Aug 2014 15:47:13 GMT\nContent-Type: text/plain\nTransfer-Encoding: chunked\nConnection: close\nSet-Cookie: __cfduid=d1983ad4f7\u2026\nLast-Modified: Fri, 27 Mar 1998 22:45:31 GMT\nETag: W/\"8982977-4c9a-32a651f0ad8c0\"\n```", "```py\n2001:0db8:85a3:0000:0000:b81a:63d6:135b\n```", "```py\n2001:db8:85a3::b81a:63d6:135b\n```", "```py\nhttp://[2001:db8:85a3::b81a:63d6:135b]/index.html\n```"]